doc/api/child_process.markdown

```diff
@@ -569,18 +305,19 @@ Example of checking for failed exec:
 
 #### options.detached
 
-On Windows, this makes it possible for the child to continue running after the
-parent exits. The child will have a new console window (this cannot be
-disabled).
+On Windows, setting `options.detached` to `true` makes it possible for the
+child process to continue running after the parent exits. The child will have
+its own console window. *Once for a child process, this cannot be disabled*.
```

Comment:
It's supposed to have been, `Once enabled for a child process, it cannot be disabled`